features/step_definitions/publication_steps.rb

```diff
@@ -35,13 +35,10 @@
   end
 end
 
-When(/^I draft a new publication "([^"]*)" relating it to the policies "([^"]*)" and "([^"]*)"$/) do |title, first_policy, second_policy|
-  begin_drafting_publication(title)
-  click_button "Save and continue"
-  click_button "Save and review legacy tagging"
-  select first_policy, from: "Policies"
-  select second_policy, from: "Policies"
-  click_button "Save"
+When(/^I visit the list of publications$/) do
+  stub_content_item_from_content_store_for(publications_path)
+  visit homepage
+  click_link "Publications"
 end
```

Comment:
@cbaines this test

Reply:
Good spot, I think this change is a result of a bad rebase. This step definition was removed in 79e3bf6eb4bac18c40e186f5edd3d718bb0e969e.